keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DockerArtifactSupplier.kt

```diff
@@ -70,10 +70,11 @@ class DockerArtifactSupplier(
     // Note: we currently don't have a way to derive account information from artifacts,
     // so, in the calls to clouddriver below, we look in all accounts.
     val latestTag = runWithIoContext {
+      // FIXME: in order to sort by properties other than version below, the service needs to return additional data
```

Comment:
Example of where we're calling an external system to retrieve artifact "versions", and will need more data in order to use the `SortingStrategy`.